### PR TITLE
Add task-based database and token management instructions for Dedicated

### LINKED ISSUE #5083: Add new admin task docs in Dedicated
`influxctl` v2.0.0 has changed and added features that need to be documented.

- Manage users /
  - Invite a user
  - List users
  - Delete a user
- Manage clusters /
   - View cluster information 
- Manage databases /
  - Update a database
- Manage tokens /
  - Update a tokens

There are also significant changes to the profile connection configuration file (`config.toml`) that may need more documentation.

##### Relevant URLs
- [_Provide relevant URLs (issues, community threads, existing doc pages, etc.)_](https://docs.influxdata.com/influxdb/cloud-dedicated/admin/)

### PR DESCRIPTION
**Pull request by sanderson**
Closes #5083 

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/admin/databases/update.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+In InfluxDB 1.x, data is stored in [databases](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#database)
+and [retention policies](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#retention-policy-rp).
+In InfluxDB Cloud Dedicated, databases and retention policies have been merged into
+_databases_, where databases have a retention period, but retention policies
+are no longer part of the data model.
+Because InfluxQL uses the 1.x data model, a database must be mapped to a v1
+database and retention policy (DBRP) to be queryable with InfluxQL.
+
+**When naming a database that you want to query with InfluxQL**, use the following
+naming convention to automatically map v1 DBRP combinations to a database:
```
Comment (powersj) [inline content/influxdb/cloud-dedicated/admin/databases/update.md:111]: I realize this section is already under create, but I have questions:

* This reads like I have to have a `/retention-policy` suffix on the database name to use influxql. Is that right?
* How does someone know what retention policy name to use? Are these figured out automatically somehow?

**Review by powersj — APPROVED** [review]
Thank you for doing this! I did leave one comment with some questions around the database naming with respect to influxql + retention policies

File: content/influxdb/cloud-dedicated/admin/databases/update.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+Database names must adhere to the following naming restrictions:
+
+- Must contain two or more characters
+- Cannot start with an underscore (`_`)
+- Cannot contain whitespace characters, double quotes (`"`), or percent signs (`%`)
+
+## InfluxQL DBRP naming convention
+
+In InfluxDB 1.x, data is stored in [databases](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#database)
+and [retention policies](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#retention-policy-rp).
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/databases/update.md:103]: Still links to v1.3, which differs from and is more helpful than what we have in `/influxdb/cloud-dedicated/reference/glossary/#retention-policy-rp`.
We should also:
-  fix the broken formatting in /influxdb/v1.3/query_language/database_management/#retention-policy-management
- port it over(?)

File: content/influxdb/cloud-dedicated/admin/tokens/update.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
+---
+title: Update a token
+description: >
+  Use the [`influxctl token update` command](/influxdb/cloud-dedicated/reference/cli/influxctl/token/update/)
+  to update a database token's permissions in your InfluxDB Cloud Dedicated cluster.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/tokens/update.md:5]: Oh. That's new.

**Review by jstirnaman — APPROVED** [review]
Suggestion on RP, but might warrant a separate fix.

File: content/influxdb/cloud-dedicated/admin/databases/update.md (hunk 2 above)
Comment (sanderson) [reply content/influxdb/cloud-dedicated/admin/databases/update.md:103]: Fixed the `latest` shortcode (it should've been going to 1.8, not 1.3) and ported the 1.x RP definition to both dedicated and serverless.

**Pull request merged**